app/controllers/qr_codes_controller.rb

```diff
@@ -29,10 +29,19 @@ def show
     end
   end
   
+  def listings
+    if employee = Employee.find_by(id: params[:employee_id])
+      #if employee.auth == params[:auth] && employee.auth_expires_at >= Time.current 
+        # Check to make sure that authorization token is current, and that the token isn't expired.
+        qr_codes = QrCode.where("company_id = " + employee.employer_id.to_s)
```

Comment:
I agree. I found a stack overflow question that advocated raw SQL statements, but I think the use-case was pretty noticeably different from mine.